CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java

```diff
@@ -44,6 +46,7 @@
  */
 @ServiceProvider(service = Plugin.class)
 @NbBundle.Messages("SaveGraphPlugin=Save Graph")
+@PluginInfo(pluginType = PluginType.UPDATE, tags = {"MODIFY"})
```

Comment:
Similarly I was thinking `EXPORT` as its saving to a file.

Reply:
Agree

Reply:
#1259